test_app/lib/screens/test_buttons/conference_service_test_buttons.dart

```diff
@@ -33,7 +33,6 @@ class ConferenceServiceTestButtons extends StatelessWidget {
         SecondaryButton(text: 'Stop video', onPressed: () => stopVideo(context)),
         SecondaryButton(text: 'Start screen share', onPressed: () => startScreenShare(context)),
         SecondaryButton(text: 'Stop screen share', onPressed: () => stopScreenShare(context)),
-        SecondaryButton(text: 'Set my spatial position', onPressed: () => setMySpatialPosition(context)),
```

Comment:
This need to be added to the context menu for each of the users.

Reply:
I added the button to the remote participant's context menu